### PR TITLE
[Synthetics] Add missing follow redirects option for multi step requests

### PR DESCRIPTION
**Pull request by romainberger**
Add support for missing option to "follow redirects" in Synthetics multi step tests.

Fixes https://github.com/DataDog/terraform-provider-datadog/issues/1077

### CODE REVIEW

File: docs/resources/synthetics_test.md [hunk 1]
```diff
@@ -427,6 +427,7 @@ Optional:
 - **body** (String) The request body.
 - **dns_server** (String) DNS server to use for DNS tests (`subtype = "dns"`).
 - **dns_server_port** (Number) DNS server port to use for DNS tests.
+- **follow_redirects** (Boolean) For API HTTP test, whether or not the test should follow redirects.
```
Comment (alai97) [inline docs/resources/synthetics_test.md:430]:
```suggestion
- **follow_redirects** (Boolean) Determines whether or not the API HTTP test should follow redirects.
```

File: datadog/resource_datadog_synthetics_test_.go [hunk 2]
```diff
@@ -788,6 +785,14 @@ func syntheticsAllowInsecureOption() *schema.Schema {
 	}
 }
 
+func syntheticsFollowRedirectsOption() *schema.Schema {
+	return &schema.Schema{
+		Description: "For API HTTP test, whether or not the test should follow redirects.",
```
Comment (alai97) [inline datadog/resource_datadog_synthetics_test_.go:790]:
```suggestion
		Description: "Determines whether or not the API HTTP test should follow redirects.",
```

**Review by alai97 — APPROVED** [review]
Looks good pending these two copy suggestions!

**Review by therve — APPROVED** [review]

**Pull request merged**